game_board/src/Main.java

```diff
@@ -1,5 +1,10 @@
+import fr.lecampusnumerique.cda2025.javaalgo.boardgames.PlayerTurns.PlayerTurn;
+import fr.lecampusnumerique.cda2025.javaalgo.boardgames.PlayerTurns.PlayerTurns;
+
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        PlayerTurns playerTurns = new PlayerTurn();
+        playerTurns.getPosition();
```

Comment:
If I well remember, we were advised to create a separate class MainDebug to test our code and leave Main alone :)